src/html/boldOrItalic.js

```diff
@@ -1,10 +1,126 @@
 'use strict';
 
 var strings = require('../strings');
-var wrapping = require('./wrapping');
 
-function boldOrItalic (chunks, type) {
-  wrapping(type === 'bold' ? 'strong' : 'em', strings.placeholders[type], chunks);
+function boldOrItalic(chunks, type) {
+  wrappingBoldItalic(type === 'bold' ? 'strong' : 'em', strings.placeholders[type], chunks);
```

Comment:
Can you link us to the general `wrapping()` function for comparison? Thanks!

Reply:
https://github.com/jywarren/woofmark/blob/master/src/html/wrapping.js